ui/component/livestreamScheduledInfo/view.jsx

```diff
@@ -3,7 +3,7 @@
 import React from 'react';
 import * as ICONS from 'constants/icons';
 import Icon from 'component/common/icon';
-import moment from 'moment';
+import moment from 'moment/min/moment-with-locales';
```

Comment:
I think `moment`'s locale was considered before and it's one of the biggest external modules, videojs aside. 

<img width="526" alt="image" src="https://user-images.githubusercontent.com/64950861/224135078-2466fbb4-8162-434e-9c2b-e987ba367c9e.png">

- Wasn't able to lazy load it in the past, but ChatGPT says it can, so maybe worth a try.
- Otherwise (for future reference), maybe https://github.com/lbryio/lbry-desktop/pull/7565 can be ported over and expanded to include `Intl.DateTimeFormat` as a lighter-weight solution.

Reply:
noticed the same a couple weeks ago, and think I brought it up to Raphael. Definitely should be on the list to try to lazy load or chunk up.